.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Issue report
```

Comment:
What's the difference between the bug report and the issue report? Shall we just keep the more generic issue report template?

Reply:
Yes I meant to only keep the issue term.